File: src/types/subscriptionPlan.ts
export type PlanTier = "basico" | "estandar" | "premium";

export interface SubscriptionPlanFeature {
  id: string;
  label: string;
  value: string;
}

export interface SubscriptionPlan {
  id: string;
  name: string;
  tier: PlanTier;
  description: string;
  price: number;
  activeProjects: string;
  revisions: string;
  features: SubscriptionPlanFeature[];
  active: boolean;
}

File: backend/app/schemas/subscription_plan.py
from pydantic import BaseModel


class SubscriptionPlanFeature(BaseModel):
    id: str
    label: str
    value: str


class SubscriptionPlan(BaseModel):
    id: str
    name: str
    tier: str
    description: str
    price: float
    activeProjects: str
    revisions: str
    features: list[SubscriptionPlanFeature]
    active: bool
